patientsearch/src/js/components/PatientListTable.js

```diff
@@ -264,7 +265,7 @@ export default function PatientListTable(props) {
       const searchResponse = searchResult.value;
       const tokenResponse = tokenResult.value;
 
-      if (searchResponse.status === 500) {
+      if (!searchResponse.ok || (searchResponse.status === 500)) {
```

Comment:
I think we want to catch any non-OK (`200`) responses. @pbugni ?
```suggestion
      if (!searchResponse.ok || (searchResponse.status !== 200)) {
```